src/snapred/ui/view/reduction/ReductionView.py

```diff
@@ -1,20 +1,22 @@
 from qtpy.QtCore import Signal
-from qtpy.QtWidgets import QHBoxLayout, QLineEdit, QPushButton, QTextEdit, QVBoxLayout, QWidget
+from qtpy.QtWidgets import QHBoxLayout, QLineEdit, QPushButton, QTextEdit, QVBoxLayout
+from snapred.backend.log.logger import snapredLogger
 from snapred.meta.decorators.Resettable import Resettable
-from snapred.ui.widget.LabeledCheckBox import LabeledCheckBox
-from snapred.ui.widget.LabeledField import LabeledField
-from snapred.ui.widget.SampleDropDown import SampleDropDown
+from snapred.ui.view.BackendRequestView import BackendRequestView
 from snapred.ui.widget.Toggle import Toggle
 
+logger = snapredLogger.getLogger(__name__)
+
 
 @Resettable
-class ReductionView(QWidget):
+class ReductionView(BackendRequestView):
     signalRemoveRunNumber = Signal(int)
 
     def __init__(self, pixelMasks=[], parent=None):
```

Comment:
`__init__` doesn't have a `pixelMasks` arg anymore.

Reply:
Why not? Should we follow the pattern or move away from it within Reduction? I'm genially unsure about the best practice here. Hopefully you have some idea?

Reply:
Okay! This has been fixed!